openapi-codegen/core/src/main/scala/sttp/tapir/codegen/RootGenerator.scala

```diff
@@ -30,13 +30,13 @@ object StreamingImplementation extends Enumeration {
   type StreamingImplementation = Value
 }
 
-object BasicGenerator {
+object RootGenerator {
```

Comment:
Renamed this because basic generator implies the existence is another more advanced generator and, uh, there's isn't one